README.md

```diff
@@ -160,6 +160,27 @@ This SDK provides a basic serialization for request/response objects but also fo
     }
     ```
 
+#### How to provide custom grpc/http client for DaprClient
+
+When building dapr client, following api will allow you to provide your own managed grpc channel within the DaprClient. This allows users to define/maintain their own thread pools and tune grpc parameters.
+Similarly you can override http client while building dapr client and tune http client parameters.
+
+   * Override custom grpc managed channel for DaprClient
+
+    ```java
+        DaprClientBuilder daprClientBuilder = new DaprClientBuilder();
+        ManagedChannelBuilder managedChannelBuilder = Grpc.newChannelBuilder("localhost:8000", TlsChannelCredentials.create())
```

Comment:
This is interesting. Which are the most common settings for a managed channel? Can this be done via additional java properties?

Reply:
Channel has everything that matters and as I know we cannot control them via out of the box sys properties. Example using my own managed executor service